src/applications/ivc-champva/10-10D/selectors/feature-toggles.js

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect';
+import { toggleValues } from '@department-of-veterans-affairs/platform-site-wide/selectors';
+import FEATURE_FLAG_NAMES from '@department-of-veterans-affairs/platform-utilities/featureFlagNames';
+
+const selectFeatureToggles = createSelector(
+  state => ({
+    isLoadingFeatureFlags: state?.featureToggles?.loading,
+    isBrowserMonitoringEnabled: toggleValues(state)[
+      FEATURE_FLAG_NAMES.form1010dBrowserMonitoringEnabled
+    ],
+  }),
+  toggles => toggles,
+);
+
+const makeSelectFeatureToggles = () => selectFeatureToggles;
+
+export { makeSelectFeatureToggles };
```

Comment:
Based on existing RUM config examples in vets-website. Creates a redux selector so these extra bits of state can be derived rather than directly stored.